app/uk/gov/hmrc/cdsimportsddsfrontend/domain/package.scala

```diff
@@ -16,7 +16,18 @@
 
 package uk.gov.hmrc.cdsimportsddsfrontend
 
+import scala.xml.{Elem, Node, Text}
+
 package object domain {
   type Eori = String
   type ConversationID = String
+
+
+  def maybeElement(name: String, maybeValue: Option[String]): Option[Node] = {
```

Comment:
Same point as above. I don't think we should be putting XML rendering code (or anything technology related) in the domain. The domain should be pure business logic

Reply:
el'll move these to the xml helpers thing

Reply:
Done. Created a type class for XmlWriters.